src/main/java/pages/OrderConfirmationPage.java

```diff
@@ -0,0 +1,36 @@
+package pages;
+
+import org.openqa.selenium.By;
+
+public class OrderConfirmationPage {
+    // Locators:
+    public By orderConfirmationMessageLocator = By.xpath("//h1[contains(text(), 'YOUR ORDER HAS BEEN PROCESSED')]");
```

Comment:
I don't recommend searching for xpaths by string.

Reply:
@Fishgame1 As discussed on the call, locators were amended to be more stable